FontSize.cpp

```diff
@@ -0,0 +1,9 @@
+#include "FontCommon.h"
+#include "FontSize.h"
+
+
+FontSize::operator unsigned int(void) const
+{
+    const unsigned int dpi = 125;
```

Comment:
At some point, this needs to be refactored to be given information about the display's DPI itself, instead of being hard-coded.